src/plugins/data/common/search/search_source/create_search_source.ts

```diff
@@ -52,10 +53,17 @@ import { SearchSourceFields } from './types';
  * @public */
 export const createSearchSource = (
   indexPatterns: IndexPatternsContract,
+  queryStringService: QueryStringContract,
   searchSourceDependencies: SearchSourceDependencies
```

Comment:
we should consider adding whatever service we end up adding to the search source dependencies

Reply:
@kavilla Added the service to the SearchSourceDependencies